tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -53,6 +65,31 @@ describe('<AddonMeta>', () => {
       });
       expect(getUserCount(root)).toMatch(/^1\.000/);
     });
+
+    it('links to stats if add-on author is viewing the page', () => {
```

Comment:
Do we have the opposite test case somewhere?

Reply:
Good catch, we didn't. I added one.